contrib/pkg/createcluster/create.go

```diff
@@ -118,13 +118,18 @@ const (
 // NewCreateClusterCommand creates a command that generates and applies cluster deployment artifacts.
 func NewCreateClusterCommand() *cobra.Command {
 	var homeDir = "."
+	var defaultPullSecretFile string
+
 	if u, err := user.Current(); err == nil {
 		homeDir = u.HomeDir
 	}
 	defaultSSHPublicKeyFile := filepath.Join(homeDir, ".ssh", "id_rsa.pub")
-	defaultPullSecretFile := filepath.Join(homeDir, ".pull-secret")
 	defaultAWSCredsFile := filepath.Join(homeDir, ".aws", "credentials")
 
+	if _, err := os.Stat(filepath.Join(homeDir, ".pull-secret")); err == nil {
```

Comment:
if `!os.IsNotExist(err)`, then we should warn the user that the pull secret file will not be used even though it exists.